package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@soos-io/api-client",
-  "version": "0.2.48",
+  "version": "0.2.49",
```

sure looks like 0.3 to me! :)
```suggestion
  "version": "0.3.0",
```